force-app/main/default/lwc/apiCallButton/apiCallButton.html

```diff
@@ -7,11 +7,13 @@
             </template>
             <template if:true={showCSPs}>
                 <lightning-formatted-text class="slds-p-left_medium" value="Success! "></lightning-formatted-text>
-                <a href={cspRelatedListUrl}>Click here to see new and existing Church School Partnerships.</a>
+                <br>
+                <a href=https://lovinghouston--partial.lightning.force.com/lightning/r/Dashboard/01Z3l000001IG0fEAG/view>Click here to see new and existing Church School Partnerships.</a>
```

Comment:
@sungysang I hard coded this but by any chance would you be able to make this dashboard link dynamic?